lib/dogma/rules/hard_tabs.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Dogma.Rules.HardTabs do
+  @moduledoc """
+  Requires that all indentation is done using spaces rather than hard tabs.
+
+  So the following would be invalid:
+
+  def something do
+  \t:body
```

Comment:
Ah, I wondered how you got a tab into the documentation without triggering the linter.